public/script.js

```diff
@@ -67,21 +95,14 @@ async function animationLoop() {
     if (!STATE.stop) {
       nextDate();
       [map, projection] = await getState(map, projection);
-      let sourceType = 'monthly';
-      if (STATE.dateLoopStyle == 'daily') {
-        sourceType = 'daily';
-      }
-      const wmsParams = {
-        LAYERS: 'NSIDC:g02135_' + STATE.extCon + `_raster_${sourceType}_` + STATE.norSouth,
-        SRS: getLocationParams().srs,
-        BBOX: getLocationParams().locationVal,
-        TILED: false,
-        format: 'image/png',
-        TIME: STATE.current.format('YYYY-MM-DD'),
-        STYLES: 'NSIDC:g02135_' + STATE.extCon + '_raster_basemap',
-      };
+      const wmsParams = getWMSParams();
       STATE.rate = 2000 - $('#speedSlider').val();
       await updateWMSLayerParams(map.getLayers().getArray()[0], wmsParams);
+      if (!validDate()) {
+        setMapText('No Data');
```

Comment:
Add the "No Data" text over an image of the map view without data instead of just a black screen.
ftp://sidads.colorado.edu/DATASETS/NOAA/G02135/north/daily/images/1978/11_Nov/N_19781108_conc_v3.0.png

Reply:
Likely just use static images for both hemispheres.